tests/python/relay/test_external_codegen.py

```diff
@@ -17,7 +17,9 @@
 """Unit tests for graph partitioning."""
```

Comment:
The changes in this file relate more to adding aot support rather than the hooks explicitly. You could consider breaking this out into a separate PR.

Reply:
Raised https://github.com/apache/tvm/pull/8591